Parse lyrics lines with timestamps 3+ digit minutes

mikooomich opened the pull request:
```
[99:34:73] Lyric test one two three
[101:34:73] Four five six
```

Would previous read as:
```
Lyric test one two three
[101:34:73] Four five six
```

With this change, the timestamp for lyrics with 3+ digits for minutes should be read correctly.

mikooomich committed:
Parse lyrics with 3+ digit minute time stamps

mikooomich committed:
Unit tests for lyrics with 3+ digit minutes

nift4 commented on app/src/test/java/org/akanework/gramophone/LrcTestData.kt:
```diff
@@ -314,8 +314,8 @@ object LrcTestData {
 		LyricLine(start = 186760uL, text = """繋げよう""", words = null, speaker = null, end = 189799uL, isTranslated = false),
-		LyricLine(start = 186760uL, text = """Let's connect""", words = null, speaker = null, end = 189799uL, isTranslated = true),
-		LyricLine(start = 189800uL, text = """""", words = null, speaker = null, end = 9223372036854775807uL, isTranslated = false),
+		LyricLine(start = 6186760uL, text = """Let's connect""", words = null, speaker = null, end = 189799uL, isTranslated = true),
```
why is end still 189799uL

mikooomich commented:
My bad, tests should be good now. I moved the lrc tests to LrcUtilsTests file, and just replaced the final chorus for ttml with "timestamp +100 minutes". Lmk if anything else is needed
